doc/active_docs/account_management_api.json

```diff
@@ -9829,11 +9820,15 @@
           "first_daily_traffic_at": {
             "type": "string",
             "description": "Timestamp of the first call on the last day when traffic was registered for the application ('Traffic On')."
+          },
+          "user_key": {
+            "type": "string",
+            "description": "User Key (API Key) of the application to be updated.",
+            "x-data-threescale-name": "user_keys"
```

Comment:
This `x-data-threescale-name` is not relevant here:
1. It does not work in the request body (only for URL params)
2. This is to specify a custom User Key for the application, we are not interested in setting it to a value that already exists. 

So this line needs to be removed.